<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaAjustes</class>
 <widget class="QMainWindow" name="VentanaAjustes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ajustes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
background-image: url(:/resources/Imagenes/fondoAjustes.png);
}

QPushButton{
background-color: rgb(153, 153, 153);
border-radius:15px;
border:1px solid;
border-color: rgb(102, 102, 102);
color:#ffffff;
font-size:13px;
}

QPushButton:hover{
background-color: rgba(16, 128, 128, 240);
}</string>
   </property>
   <widget class="QPushButton" name="botonSave">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>480</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonCancelar">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>480</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLabel" name="AnchMax">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Anchura máxima (px): </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="campoAnchura">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>1920</string>
    </property>
   </widget>
   <widget class="QLabel" name="AltMax">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Altura máxima (px):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="campoAltura">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>1080</string>
    </property>
   </widget>
   <widget class="QLabel" name="framerate">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Framerate máximo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="bitrate">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Bitrate máximo (Mbps):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="campoFramerate">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>60</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="campoBitrate">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="activarInfo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>405</y>
      <width>441</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Mostrar información de rendimiento del video durante la reproducción</string>
    </property>
   </widget>
   <widget class="QLabel" name="dibujoAjustes">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans</family>
      <pointsize>37</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Ajustes</string>
    </property>
   </widget>
   <widget class="QLabel" name="dibujoTiempo">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>334</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/Imagenes/cronometro.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dibujoAltura">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>190</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/Imagenes/alto.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dibujoFPS">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>270</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/Imagenes/fps.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dibujoAnchura">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/resources/Imagenes/ancho.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
